### DISCUSSION TITLE
why relases 2.1.0 has no kvrocks-xxx-release.x86_64.rpm file ?

### DISCUSSION THREAD

**Discussion by corningsun** [apache/kvrocks#779] [Q&A] [answered]
why relases 2.1.0 has no kvrocks-xxx-release.x86_64.rpm file ?

**Comment by git-hulk** [comment, marked as answer]
Hello @corningsun 

We need to vote those binaries before releasing in Apache community, and this may block our first release. To make our first release in Apache incubator easier, we decided to release source code only. You can build the binary and feedback if have any question.

**Reply by corningsun** [reply]
I build success with modify the **x.py** file

`cmake_options += ["-DENABLE_STATIC_LIBSTDCXX=OFF", "-DWITH_LZ4=ON", "-DWITH_ZLIB=ON", "-DWITH_SNAPPY=ON"]`

sysinfo

```
# cat /proc/version
Linux version 3.10.0-1127.19.1.el7.x86_64 (mockbuild@kbuilder.bsys.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-39) (GCC) ) #1 SMP Tue Aug 25 17:23:54 UTC 2020
```

bug got error when I set a key to kvrocks

```
eg: hset KEY_192.168.1.1 888

ERR Corruption: Corrupted compressed block contents: Snappy

```

**Reply by git-hulk** [reply]
You can submit a new issue and paste how to reproduce steps, coz the above issue didn't relate to the current topic.

**Reply by PragmaTwice** [reply]
Hi @corningsun , what error did you encounter with the default build options? i.e. `./x.py build <dir>`

BTW, you can pass cmake options directly to the script, e.g. `./x.py build <dir> -DX=1 -DY=2`, without modifying it.
